pkg/runtime/engine/apptainer/config/config.go

```diff
@@ -498,6 +499,16 @@ func (e *EngineConfig) GetNoCwd() bool {
 	return e.JSON.NoCwd
 }
 
+// SetSkipBinds sets bind paths to skip
+func (e *EngineConfig) SetSkipBinds(val []string) {
+	e.JSON.SkipBinds = val
+}
+
+// GetSkipBinds gets bind paths to skip
+func (e *EngineConfig) GetNoBinds() []string {
```

Comment:
Please rename to `GetSkipBinds` for consistency as it is also set in function comment

Reply:
I renamed function and parameter for consistency. Is it ok?

Reply:
:+1: thanks @edytuk !